src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -1300,6 +1300,42 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                 GetEmitter()->emitIns_R_PATTERN(ins, emitSize, targetReg, opt, SVE_PATTERN_ALL);
                 break;
 
+            case NI_Sve_CreateWhileLessThanMask8Bit:
+            case NI_Sve_CreateWhileLessThanOrEqualMask8Bit:
```

Comment:
Is this going to be a more common pattern? Should we have a way to make it more table driven?

Reply:
Problem is we need to know the name of the intrinsic in order to know the `opt` value (eg `INS_OPTS_SCALABLE_H`).

Reply:
Are we not tracking the vector type anywhere? We have a couple fields (simdBaseJitType and altType for example) so we should be able to track both the overload type (`int` vs `uint` vs `long` vs `ulong`) and the vector base type/size